.travis.yml

```diff
@@ -32,10 +32,15 @@ env:
   - VERSION="10.0" TESTS="0" LINT_CHECK="0" TRANSIFEX="0"
 
 install:
+  - pip install ezodf==0.3.2
```

Comment:
@sudhir-serpentcs To modify travis better on a new PR. 

@seb-elico ?

Reply:
Oh, yes. My intention was just to make travis green.
@victormartinelicocorp @seb-elico

Reply:
It is ok: Let's fix everything on this PR and move quick